daemon/snapper/snapper.go

```diff
@@ -36,7 +38,11 @@ type snapProgress struct {
 	doneAt time.Time
 
 	// SnapErr
-	err error
+	errs struct {
+		preHook  []error
```

Comment:
These should be the new `*snapper.Error` type I proposed (See `RunHookCommand`).
The piece of context that's missing to render `snapProgress` with `errs.preHook != nil` into a proper error message that explains to the user what hook failed why is the information **which** hook in the configuration produced the error.
Perhaps we should give names to the hooks in the configuration, and include the hook name in `snapper.Error`?